src/main/service/manualUploadService.ts

```diff
@@ -46,6 +47,18 @@ export class ManualUploadService {
         publicationService.getListTypes().forEach((value, key) => {
             jsonArray.push({ value: key, text: value.shortenedFriendlyName });
         });
+        jsonArray.push({ value: 'EMPTY', text: '<Please choose a list type>' });
+        jsonArray.sort((a, b) => {
```

Comment:
This could be simplified to one line make it easier to read:
jsonArray.sort((a,b) =>  (a.text.toUpperCase() > b.text.toUpperCase() ? 1 : -1));

Reply:
Yeah! Have done this now :)